featuretools/synthesis/deep_feature_synthesis.py

```diff
@@ -1276,6 +1262,26 @@ def _features_have_same_path(input_features):
     return True
 
 
+def _build_ignore_columns(input_dict: Dict[str, List[str]]) -> DefaultDict[str, set]:
+    """Iterates over the input dictionary and builds the ignore_columns data structure"""
+    ignore_columns = defaultdict(set)
+    if input_dict is not None:
+        for df_name, cols in input_dict.items():
+            _validate_ignore_columns_entry(df_name, cols)
```

Comment:
if `_build_ignore_columns` is now separated out, should we move the code for `_validate_ignore_columns_entry` back in?

Reply:
I think that makes sense